src/js/script.js

```diff
@@ -90,13 +111,40 @@ function audioSetup() {
       }
     }
   }, false);
-  audioFile.play();
 
-  // Make music playable on some mobile devices
-  const ts = document.addEventListener('touchstart', function() {
-    audioFile.play();
-    document.removeEventListener('touchstart', ts);
-  });
+  function playUnmuteAudio() {
+    audioFile.muted = false;
+
+    // The muted play might have failed, then play it now.
+    if (audioFile.paused) {
+      /*
+        Note: For some reason Firefox still might fail here, but only on key
+        events if a special key is pressed (e.g. Control, Shift, etc).
+      */
+      audioFile.play();
```

Comment:
We should avoid uncaught Exceptions?

Reply:
We should not remove the listeners if firefox (or other browsers) fails?

Reply:
Good point, probably not. 🙂